app/constantData/userRoleOption.ts

```diff
@@ -0,0 +1,6 @@
+const PROFESSION_OPTIONS = [
```

Comment:
Use readonly for Type Safety:
Ensures the array and its objects are immutable.
```suggestion
const PROFESSION_OPTIONS: readonly ProfessionOption[] = [
```

Reply:
made it read only